### PR TITLE
Proposal: add docs about integrations

### PR DESCRIPTION
**Pull request by maorfr**
to make qontract-reconcile and app-interface truly open source, we need to add docs on usage.

this is the first PR (out of hopefully many) adding such docs.

### CODE REVIEW

File: reconcile/cli.py [hunk 1]
```diff
@@ -1458,11 +1458,11 @@ def gabi_authorized_users(ctx, thread_pool_size, internal, use_jump_host):
...
 
-@integration.command()
-@enable_deletion(default=False)
-@click.pass_context
-def dyn_traffic_director(ctx, enable_deletion):
-    run_integration(reconcile.dyn_traffic_director, ctx.obj, enable_deletion)
+# @integration.command()
+# @enable_deletion(default=False)
+# @click.pass_context
+# def dyn_traffic_director(ctx, enable_deletion):
```
Comment (fishi0x01) [inline reconcile/cli.py:1464]: Is removing this intended? Not mentioned in the PR description

File: README.md [hunk 2]
```diff
@@ -46,20 +46,20 @@ Additional tools that use the libraries created by the reconciliations are also
 - `kafka-clusters`: Manages Kafka clusters via OCM.
 - `ldap-users`: Removes users which are not found in LDAP search.
 - `openshift-clusterrolebindings`: Configures ClusterRolebindings in OpenShift clusters.
-- `openshift-groups`: Manages OpenShift Groups.
+- `openshift-groups`: [Manages OpenShift Groups.](/docs/integreations/openshift_groups.md)
```
Comment (rporres) [inline README.md:49]: while I love the word `integreations` as it is indeed, great, this should be `integrations`

Comment (maorfr) [reply README.md]: oops!

File: reconcile/cli.py (hunk 1 above)
Comment (maorfr) [reply reconcile/cli.py]: no no, it was a mistake

File: docs/integrations/openshift_network_policies.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Enable network traffic between Namespaces via App-Interface (`/openshift/namespace-1.yml`)
+
+To enable network traffic between namespaces, a user must add the following to a namespace declaration:
```
Comment (geoberle) [inline docs/integrations/openshift_network_policies.md:3]: in line 10 it is correctly stated that the traffic is allowed from the source namespace to "this" namespace, but this line suggests something different with the wording "between"

**Review by geoberle — APPROVED** [review]
thank you for starting this!

**Comment by maorfr** [comment]
@patjlm could this PR inspire https://github.com/app-sre/qontract-docs?

**Pull request closed without merging**